### PR TITLE
✨PRTL-2371 default top mutated adjustments

### PR DESCRIPTION
**Pull request by anncatton**
## Ticket Number

PRTL-2371

## Environment to be used in testing

- [x] `prod`
- [x] `dev-oicr`
- [ ] `qa` (which version?)

## Description of Changes
Deletes legacy ProjectsCharts component.
Replaces explore CaseAggregations with modern component.
Sets default query filters on all explore links that do not have other filters on these components:
 - Clinical analysis launch
 - Select Set
 - Header link
 - Home exploring links
 - Manages sets
 - Projects page -> charts.

### CODE REVIEW

File: src/packages/@ncigdc/components/SearchPage.js [hunk 1]
```diff
@@ -57,49 +57,52 @@ const enhance = compose(
...
-        </Row>
-        {results}
-      </Content>
-    </Container>
+          />
+      </FacetsPanel>
+    )}
+    <Content>
+      <Row style={{ marginBottom: '2rem' }}>
+        {!showFacets && (
```
Comment (samrichca) [inline src/packages/@ncigdc/components/SearchPage.js]:
```suggestion
        {showFacets || (
```

??

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 2]
```diff
@@ -245,45 +243,48 @@ const CurrentFilters = (
...
+                    </NotUnderlinedLink>
                 ))}
                 {value.length > 2 && (
                   <UnstyledButton
                     className="test-toggle"
-                    style={styles.rightParen}
                     onClick={() => onLessClicked(filter)}
-                  >
+                    style={styles.rightParen}
+                      >
```
Comment (caravinci) [inline src/packages/@ncigdc/components/CurrentFilters.js]:
```suggestion
                    >
```

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 3]
```diff
@@ -245,45 +243,48 @@ const CurrentFilters = (
...
                   <UnstyledButton
                     className="test-toggle"
-                    style={{ display: 'flex', alignItems: 'center' }}
                     onClick={() => onLessClicked(filter)}
-                  >
+                    style={{
+                        display: 'flex',
+                        alignItems: 'center',
+                      }}
+                      >
```
Comment (caravinci) [inline src/packages/@ncigdc/components/CurrentFilters.js]:
```suggestion
                    >
```

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 4]
```diff
@@ -245,45 +243,48 @@ const CurrentFilters = (
...
                   </UnstyledButton>
                 )}
                 {isFilterExpanded(filter) && (
                   <UnstyledButton
                     className="test-toggle"
-                    style={{ display: 'flex', alignItems: 'center' }}
                     onClick={() => onLessClicked(filter)}
-                  >
+                    style={{
+                        display: 'flex',
```
Comment (caravinci) [inline src/packages/@ncigdc/components/CurrentFilters.js]:
```suggestion
                      display: 'flex',
```

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 5]
```diff
@@ -245,45 +243,48 @@ const CurrentFilters = (
...
+                        alignItems: 'center',
```
Comment (caravinci) [inline src/packages/@ncigdc/components/CurrentFilters.js]:
```suggestion
                      alignItems: 'center',
```

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 6]
```diff
@@ -245,45 +243,48 @@ const CurrentFilters = (
...
+                      }}
```
Comment (caravinci) [inline src/packages/@ncigdc/components/CurrentFilters.js]:
```suggestion
                    }}
```

File: src/packages/@ncigdc/components/analysis/SelectSet.js [hunk 7]
```diff
@@ -1,16 +1,18 @@
 // @flow
 import React from 'react';
-import _ from 'lodash';
+import {
+  capitalize, get, omit, set, truncate,
```
Comment (caravinci) [inline src/packages/@ncigdc/components/analysis/SelectSet.js]:
```suggestion
  capitalize,
  get, 
  omit, 
  set, 
  truncate,
```

File: src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js [hunk 8]
```diff
@@ -85,26 +85,26 @@ export default compose(
       props.analysisViewer.analysis.top_cases_count_by_genes.data,
     );
 
-    const caseAggs = aggregations.projects.buckets.filter(b =>
-      props.projectIds.includes(b.key),
-    );
+    const caseAggs = aggregations.projects.buckets.filter(b => props.projectIds.includes(b.key),);
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js]:
```suggestion
    const caseAggs = aggregations.projects.buckets.filter(b => props.projectIds.includes(b.key));
```

File: src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js [hunk 9]
```diff
@@ -161,38 +160,53 @@ export default compose(
...
+        }),
         tooltips: stackedBarCalculations[geneId].byProject.reduce(
-          (acc, { projectId, percent, count }) => ({
+          (acc, { count, percent, projectId }) => ({
             ...acc,
             [projectId]: (
               <span>
                 <b>
-                  {projectId}:{' '}
+                  {projectId}
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js]:
```suggestion
                  {${projectId}: `}
```

File: src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js [hunk 10]
```diff
@@ -161,38 +160,53 @@ export default compose(
...
+:
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js]:
```suggestion
```

File: src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js [hunk 11]
```diff
@@ -161,38 +160,53 @@ export default compose(
...
+                  {' '}
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js]:
```suggestion
```

File: src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js [hunk 12]
```diff
@@ -161,38 +160,53 @@ export default compose(
...
                     (projects.find(p => p.project_id === projectId) || {
                       name: '',
                     }).name
                   }
                 </b>
-                <br /> {count.toLocaleString()} Case{count > 1 ? 's' : ''}{' '}
-                Affected<br />
-                {count.toLocaleString()} / {numUniqueCases.toLocaleString()} ({percent.toFixed(2)}%)
+                <br />
+                {' '}
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js]: Wish we could automatically convert these next few lines (and others of the kind, anywhere else) into string interpolations instead.
May have to look into it at ESLint's own repo 🤔

File: src/packages/@ncigdc/containers/explore/ExplorePage.js [hunk 13]
```diff
@@ -2,6 +2,8 @@
 import React from 'react';
 import Relay from 'react-relay/classic';
 import { get, isEqual } from 'lodash';
+import { compose, lifecycle, withState } from 'recompose';
```
Comment (caravinci) [inline src/packages/@ncigdc/containers/explore/ExplorePage.js]:
```suggestion
import { 
  compose,
  lifecycle,
  withState,
} from 'recompose';
```

File: src/packages/@ncigdc/containers/explore/ExplorePage.js [hunk 14]
```diff
@@ -97,137 +99,129 @@ function setVariables({ filters, relay }) {
...
 
-  render() {
-    const {
-      filters,
-      push,
-      relay,
-      viewer,
-    } = this.props;
-    const { maxFacetsPanelHeight } = this.state;
+const enhance = compose(
```
Comment (caravinci) [inline src/packages/@ncigdc/containers/explore/ExplorePage.js]:
```suggestion
const enhance = compose(
  setDisplayName('EnhancedExplorePageComponent'),
```

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 15]
```diff
@@ -245,45 +243,48 @@ const CurrentFilters = (
...
-                      offset: 0,
-                      filters: {
-                        op: 'and',
-                        content: [
-                          {
-                            op: filter.op,
-                            content: {
-                              field: filter.content.field,
-                              value: [value],
+                      className="test-field-value"
```
Comment (caravinci) [inline src/packages/@ncigdc/components/CurrentFilters.js]: any clue why eslint would indent these and extra level in this commit?

File: src/packages/@ncigdc/containers/explore/ExplorePage.js [hunk 16]
```diff
@@ -361,12 +357,12 @@ export const ExplorePageQuery = {
             ${CaseAggregations.getFragment('facets')}
           }
           cases {
-            aggregations(filters: $filters aggregations_filter_themselves: false) {
-              ${CaseAggregations.getFragment('aggregations')}
-            }
             hits(first: $cases_size offset: $cases_offset filters: $filters score: $cases_score sort: $cases_sort) {
               total
             }
+            aggregations(filters: $filters aggregations_filter_themselves: false) {
```
Comment (caravinci) [inline src/packages/@ncigdc/containers/explore/ExplorePage.js]: is there a non-obvious need to resort these after hits?

**Pull request merged**